index.html

```diff
@@ -65,6 +65,24 @@ <h1>Rädda Bina!</h1>
         </div>
 
         <div class="why">
+            <h2 class="why-h2">Varför?</h2>
+            <div class="why-box">
+                <img class="why-img" src="img/desktop/icon-bee1.png" alt="drawn picture of a bee from the top">
+                <h3 class="why-h3">Rädda bina!</h3>
+                <p class="why-p">Bina är livsviktiga för vår matproduktion och hela ekosystemet. Men en tredjedel av våra vilda biarter är på väg att försvinna.</p>
+            </div>
+
+            <div class="why-box">
+                <img class="why-img" src="img/desktop/icon-bee2.png" alt="drawn picture of a bee from it's side">
+                <h3 class="why-h3">Livsviktiga!</h3>
+                <p class="why-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            </div>
+
+            <div class="why-box">
+                <img class="why-img" src="img/desktop/icon-bee1.png" alt="drawn picture of a bee">
```

Comment:
Also alt text here :) Screen readers often read "Image/picture of {{alt text}}", so no need to add the "Picture of" part :)